source/developers/concepts/platform_overview.rst

```diff
@@ -0,0 +1,84 @@
+Platform Overview
+#################
+
+The Open edX Platform is made up of multiple web services that work together to
+provide all the tools you might need to build and run courses online.
+
+Tech Stack
+**********
+
+While there are a few exceptions, for the most part the technology stack for the
+Open edX Platform uses `Django`_ for backend services and `React`_ for the frontend.
+There are multiple frontend apps which are usually referred to as
+MFEs (Micro-FrontEnds) within the ecosystem and community.
+
+The core code base lives on GitHub in the `openedx`_ organization.
+
+Platform Layout
+***************
+
+While the platform contains many services, MFEs and libraries the core of the
+platform lives in the `edx-platform`_ and `frontend-platform` repositories.
+
+* edx-platform - This was the first service created in the Open edX ecosystem and
+  contains the legacy monolith application. That application currently can run
+  in two modes.
+
+  As the LMS (Learning Management System) it serves as the backend
+  for learner data as they go through courses, programs, and serves some legacy
+  UI elements.
+
+  As Studio it is the CMS(Content Management System) where instructors and
+  course authors can build the course content that learners will consume.
+
+
+* frontend-platform - This is a newer core system meant to ease the creation of
+  multiple small frontend applications (MFEs) by bundling together many common
+  utilities that all frontends need.  This includes things like authentication
+  workflows, logging, monitoring and translations related tooling.
+
+Along with these two core systems, there exist a number of other `backend
+services`_ and `MFEs`_ which you can find on GitHub.
+
+Libraries
+=========
+
+In addition to the backends and front-ends mentioned above, the platform is made
+of numerous libraries, that live in the `openedx`_ organization.
+These libraries accomplish a variety of different tasks, and you can find more
+details about what they do in their READMEs.
```

Comment:
We should avoid the use of the word "you" since this is formal documentation.
```suggestion
These libraries accomplish a variety of different tasks. More details about
what they do can be found in their READMEs.
```